### ISSUE TITLE
Number or symbols

### ISSUE THREAD

**Issue by csperkins** [ietf-tapswg/api-drafts#844]
In the interface draft, section 1.1:
```
   *  Integer: Instances take positive or negative numeric integer
      values, or sometimes special non-numeric (symbolic) values.
   *  Numeric: Instances take positive or negative numeric values, or
      sometimes special non-numeric (symbolic) values.
```
Do we need to conflate numeric types and enumerations, or can numbers be numbers?

**Comment by mwelzl** [comment]
From interim: add a clarification on what this is about, why it (symbolic values) is there.

**Comment by tfpauly** [comment]
Fixed by #964
